<<<<<<< Updated upstream
[2020-04-21 19:21:55,211]{/home/andrenovado/Documents/Coding/Alterra/.virtualenvs/rest_training/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /weather/q [GET]
Traceback (most recent call last):
  File "/home/andrenovado/Documents/Coding/Alterra/.virtualenvs/rest_training/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/andrenovado/Documents/Coding/Alterra/.virtualenvs/rest_training/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/andrenovado/Documents/Coding/Alterra/.virtualenvs/rest_training/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/andrenovado/Documents/Coding/Alterra/.virtualenvs/rest_training/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/andrenovado/Documents/Coding/Alterra/.virtualenvs/rest_training/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/andrenovado/Documents/Coding/Alterra/andre_derby_restful_api/blueprints/weather/resources.py", line 31, in get
    shirt = GetPriceReport().get('shirt for winter')
  File "/home/andrenovado/Documents/Coding/Alterra/andre_derby_restful_api/blueprints/amazon/resources.py", line 35, in get
    converted_price = GetConversion().get(result_product['price'])
  File "/home/andrenovado/Documents/Coding/Alterra/andre_derby_restful_api/blueprints/conversion/resources.py", line 31, in get
    price = float((re.findall("[0-9]+,?.?[0-9]+", product_price))[0].replace(',','.'))
IndexError: list index out of range
[2020-04-21 19:21:55,215]{/home/andrenovado/Documents/Coding/Alterra/andre_derby_restful_api/blueprints/__init__.py:59} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "uri": "/weather/q?city=moscow&name=Andre&email=andrenovado@gmail.com&target_currency=GBP", "request": null, "response": {"message": "Internal Server Error"}}
[2020-04-21 19:26:29,947]{/home/andrenovado/Documents/Coding/Alterra/andre_derby_restful_api/blueprints/__init__.py:49} WARNING - REQUEST_LOG	{"method": "GET", "code": "200 OK", "uri": "/weather/q?city=manchester&name=Andre&email=andrenovado@gmail.com&target_currency=GBP", "request": null, "response": ["Your destination temperature is 7.51 \u00b0 Celcius", {"ASIN": "B0819C9C93", "title": "voofly Cycling Jersey Long Sleeve Winter Winproof Full Zipper Mountain Biking Shirts for Men Hi-viz Green Medium", "price": "$39.99", "listPrice": "$99.99", "imageUrl": "https://m.media-amazon.com/images/I/41aQeC6mEiL._SL160_.jpg", "detailPageURL": "https://www.amazon.com/dp/B0819C9C93", "rating": "4.4", "totalReviews": "21", "subtitle": "", "isPrimeEligible": "1", "converted_price": "32.14 GBP"}, {"ASIN": "B017YQL1X2", "title": "Wrangler Authentics Men's Authentics Fleece-Lined Cargo Pant, Anthracite Twill, 34W X 34L", "price": "$29.91", "listPrice": "", "imageUrl": "https://m.media-amazon.com/images/I/41I+IxftPXL._SL160_.jpg", "detailPageURL": "https://www.amazon.com/dp/B017YQL1X2", "rating": "4.6", "totalReviews": "3019", "subtitle": "Wrangler", "isPrimeEligible": "1", "converted_price": "24.04 GBP"}, {"ASIN": "B01N6IRDEU", "title": "iloveSIA Womens Hooded Coat Faux Fur Lined Jacket Black 12", "price": "$45.99", "listPrice": "", "imageUrl": "https://m.media-amazon.com/images/I/41VTV1CFd3L._SL160_.jpg", "detailPageURL": "https://www.amazon.com/dp/B01N6IRDEU", "rating": "4.4", "totalReviews": "2481", "subtitle": "iloveSIA", "isPrimeEligible": "1", "converted_price": "36.96 GBP"}, {"ASIN": "B07JN5SQSQ", "title": "Crocs Women's Classic Tie Dye Graphic Clog, White/Multi, 9 US Women / 7 US Men", "price": "$29.99", "listPrice": "", "imageUrl": "https://m.media-amazon.com/images/I/31A68kTiVcL._SL160_.jpg", "detailPageURL": "https://www.amazon.com/dp/B07JN5SQSQ", "rating": "4.8", "totalReviews": "2600", "subtitle": "", "isPrimeEligible": "1", "converted_price": "24.10 GBP"}, {"ASIN": "B07W548DGJ", "title": "YQXCC Kids Winter Hats Balaclava Ski Mask Windproof Warm Adjustable with Fleece Lining Hat for Boys Girls (Black)", "price": "$7.99", "listPrice": "", "imageUrl": "https://m.media-amazon.com/images/I/31jGtVG27+L._SL160_.jpg", "detailPageURL": "https://www.amazon.com/dp/B07W548DGJ", "rating": "4.6", "totalReviews": "355", "subtitle": "YQXCC", "isPrimeEligible": "1", "converted_price": "6.42 GBP"}]}
=======
[2020-04-21 18:47:25,501]{/home/alta8/Documents/RESTAPI/andre_derby_restful_api/blueprints/__init__.py:55} WARNING - REQUEST_LOG	{"method": "POST", "code": "200 OK", "uri": "/client?", "request": {"client_key": "client1", "client_secret": "alta123", "status": "False"}, "response": {"id": 2, "client_key": "client1", "client_secret": "d3558f2eb8b06d60d766bc5bf28cc8a4ed2fd2e42073886b3c148b055600f1285ccf1b23b27cd5096b6f45ebd9e80d6f4efaed5981619aba710e1619a44a4157", "salt": "3e8c8ac9403b49c9938c92865a5b4fe1", "status": "False", "created_at": "Tue, 21 Apr 2020 18:47:25 -0000", "updated_at": null, "deleted_at": null}}
>>>>>>> Stashed changes
[2020-04-21 22:58:53,759]{/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /client [GET]
Traceback (most recent call last):
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta8/Documents/RESTAPI/andre_derby_restful_api/blueprints/__init__.py", line 20, in wrapper
    verify_jwt_in_request()
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 32, in verify_jwt_in_request
    jwt_data, jwt_header = _decode_jwt_from_request(request_type='access')
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 314, in _decode_jwt_from_request
    raise NoAuthorizationError(errors[0])
flask_jwt_extended.exceptions.NoAuthorizationError: Missing Authorization Header
[2020-04-21 22:58:53,764]{/home/alta8/Documents/RESTAPI/andre_derby_restful_api/blueprints/__init__.py:65} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "uri": "/client?", "request": null, "response": {"message": "Internal Server Error"}}
[2020-04-21 22:59:32,445]{/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /weather/q [GET]
Traceback (most recent call last):
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta8/Documents/RESTAPI/andre_derby_restful_api/blueprints/__init__.py", line 20, in wrapper
    verify_jwt_in_request()
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 32, in verify_jwt_in_request
    jwt_data, jwt_header = _decode_jwt_from_request(request_type='access')
  File "/home/alta8/Documents/virtualenv/project_restapi/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 314, in _decode_jwt_from_request
    raise NoAuthorizationError(errors[0])
flask_jwt_extended.exceptions.NoAuthorizationError: Missing Authorization Header
[2020-04-21 22:59:32,446]{/home/alta8/Documents/RESTAPI/andre_derby_restful_api/blueprints/__init__.py:65} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "uri": "/weather/q?city=london&name=derby&email=derbyprayogo@gmail.com&target_currency=IDR", "request": null, "response": {"message": "Internal Server Error"}}
